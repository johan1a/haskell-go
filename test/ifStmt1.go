func main() {
if b { a }
}
