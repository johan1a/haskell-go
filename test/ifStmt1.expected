[IfStmt (Ifstmt1 (IdUse "b") (Block [SimpleStmt (ExpressionStmt (IdUse "a"))]))]